container/Makefile

```diff
@@ -39,9 +39,11 @@ build-deb: build
 
 .PHONY: build-base-test
 build-base-test: needslim
+	cp ../gardenlinux.asc base-test/gardenlinux.asc
```

Comment:
```suggestion
	cp -p ../gardenlinux.asc base-test/gardenlinux.asc
```

Reply:
Hey @Vincinator,
thanks for your feedback. Can you provide more information about your issue? Maybe you already have an equal issue within the `build-image` target, where this file gets copied, too. However, preserving the file attributes should be fine here. Added within [51388fa](https://github.com/gardenlinux/gardenlinux/pull/1350/commits/51388fafeab1f64932277977ad6b256d096f64a3) (squashed).

Reply:
```
W: http://deb.debian.org/debian/dists/testing/InRelease: The key(s) in the keyring /etc/apt/trusted.gpg.d/gardenlinux.asc are ignored as the file is not readable by user '_apt' executing apt-key.
W: http://deb.debian.org/debian-security/dists/testing-security/InRelease: The key(s) in the keyring /etc/apt/trusted.gpg.d/gardenlinux.asc are ignored as the file is not readable by user '_apt' executing apt-key.
W: http://deb.debian.org/debian/dists/testing-updates/InRelease: The key(s) in the keyring /etc/apt/trusted.gpg.d/gardenlinux.asc are ignored as the file is not readable by user '_apt' executing apt-key.
Reading package lists...
```

Added a line to `container/build-image/Dockerfile` to check file permissions of the gardenlinux.asc key
```
STEP 6/11: RUN ls -lah $GARDENLINUX_MIRROR_KEY
-rw-r----- 1 root root 1006 Nov 15 12:33 /etc/apt/trusted.gpg.d/gardenlinux.asc
```
But expected file permissions to be `644`